Wikipedia/Code/PageHistoryCountsViewController.swift

```diff
@@ -80,6 +80,11 @@ class PageHistoryCountsViewController: UIViewController {
 
         filterCountsViewController.delegate = self
         wmf_add(childController: filterCountsViewController, andConstrainToEdgesOfContainerView: filterCountsContainerView)
+
+        sparklineView.isAccessibilityElement = true
```

Comment:
I don't feel strongly about this, but some of the accessibility related info I read reminded me voice-over is used by non-blind users (low vision etc) - and I thought a bit of context might be nice for the graph...